packages/rich-text/src/component/index.js

```diff
@@ -214,6 +214,19 @@ export function useRichText( {
 			return;
 		}
 
+		if ( ref.current.ownerDocument.activeElement !== ref.current ) {
+			// Can't focus if there is a parent content editable.
```

Comment:
Any chance that there would be more than one such ancestor? Or that we should guard against corrupt DOM states?

Reply:
I will comment more here. We don't have any deeper nesting no :) This fix is actually needed because writing flow doesn't turn off `contenteditable` quick enough. Hooks in child components execute first, and the writing flow hooks last. I wish we had a better fix for this.